extensions/amp-action-macro/validator-amp-action-macro.protoascii

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+tags: {  # amp-action-macro
+  html_format: AMP
+  html_format: AMP4ADS
+  html_format: AMP4EMAIL
+  html_format: ACTIONS
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-action-macro"
+    version: "0.1"
+    version: "latest"
+    requires_usage: GRANDFATHERED
+    deprecated_allow_duplicates: true
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-action-macro>
+  html_format: AMP
+  html_format: AMP4ADS
+  html_format: AMP4EMAIL
+  html_format: ACTIONS
+  tag_name: "AMP-ACTION-MACRO"
+  requires_extension: "amp-action-macro"
+  attrs: { name: "id" mandatory: true }
+  attrs: { name: "execute" mandatory: true }
+  attrs: { name: "arguments" mandatory: false }
```

Comment:
please alphabetize by attrs.name, e.g. arguments first, then execute, then id.

also mandatory defaults to false so unnecessary to include it.

Reply:
done